### PR TITLE
Restore escaping for command

### PR DESCRIPTION
**Pull request by dron2004**
As I promised, I restored escaped command line and its arguments. And I made it for the Windows platform also.

### CODE REVIEW

File: src/MockWebServer.php [hunk 1]
```diff
@@ -285,27 +274,43 @@ private function isWindowsPlatform() {
...
-			$fullCmd = 'exec ' . $fullCmd;
-		}
-
 		$pipes = [];
 		$env   = null;
 		$cwd   = null;
 
 		$process = proc_open($fullCmd, [], $pipes, $cwd, $env, [
-			'suppress_errors' => false,
+			'suppress_errors' => true,
```
Comment (donatj) [inline src/MockWebServer.php]: Why are we supressing errors?

Comment (dron2004) [reply src/MockWebServer.php]: BTW it is the Windows-only option.

**Comment by donatj** [comment]
I'm pondering using symfony/process for handling the escaping of arguments. It's windows aware.

**Comment by donatj** [comment]
#42 replaced this

**Pull request closed without merging**